astroquery/jplhorizons/tests/test_jplhorizons_remote.py

```diff
@@ -4,6 +4,7 @@
 from astropy.tests.helper import remote_data, assert_quantity_allclose
 from numpy.ma import is_masked
 import numpy.testing as npt
+from astropy.units.quantity import allclose
```

Comment:
I think  these should be `assert_quantity_allclose` as `allclose()` doesn't have the assert (and neither available for older astropy versions)

Reply:
D'oh, of course! Thanks!